zjit/src/hir.rs

```diff
@@ -6145,6 +6183,27 @@ mod opt_tests {
         "#]]);
     }
 
+    #[test]
+    fn test_opt_isnil_pattern_matching() {
+        eval("def test = ([] in [])");
+
+        assert_optimized_method_hir("test",  expect![[r#"
```

Comment:
```suggestion
        assert_optimized_method_hir("test", expect![[r#"
```

nitpick: You seem to like placing two spaces here, but it seems inconsistent with the style of other test code.

Reply:
ah so sorry! I must have kept copying that over from my other tests, I'll be more mindful in the future, should no longer be on this branch now.